app/src/main/java/com/practicum/playlistmaker/media/ui/playlists/PlaylistsViewHolder.kt

```diff
@@ -9,25 +9,29 @@ import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.practicum.playlistmaker.R
 import com.practicum.playlistmaker.media.domain.models.Playlist
-import com.practicum.playlistmaker.media.domain.utils.getTrackCountNoun
-import com.practicum.playlistmaker.media.ui.fragment.NewPlaylistFragment
+import com.practicum.playlistmaker.utils.getTrackCountNoun
+import com.practicum.playlistmaker.media.ui.new_playlist.NewPlaylistFragment
 import java.io.File
 
 class PlaylistsViewHolder(parentView: ViewGroup) : RecyclerView.ViewHolder(
     LayoutInflater.from(parentView.context)
         .inflate(R.layout.yours_playlist_item, parentView, false)
 ) {
-    private val playlistImageView: ImageView by lazy { itemView.findViewById(R.id.ivPlaylistGridItem)}
-    private val playlistNameView: TextView by lazy { itemView.findViewById(R.id.tvPlaylistGridItemName)}
-    private val playlistTrackInfoView: TextView by lazy { itemView.findViewById(R.id.tvPlaylistGridItemTrackInfo)}
+    private val playlistImageView: ImageView by lazy { itemView.findViewById(R.id.ivPlaylistGridItem) }
+    private val playlistNameView: TextView by lazy { itemView.findViewById(R.id.tvPlaylistGridItemName) }
+    private val playlistTrackInfoView: TextView by lazy { itemView.findViewById(R.id.tvPlaylistGridItemTrackInfo) }
 
     fun bind(model: Playlist) {
         playlistNameView.text = model.name.toString()
-        playlistTrackInfoView.text = String.format("%d %s", model.trackCount, getTrackCountNoun(model.trackCount))
+        playlistTrackInfoView.text =
+            String.format("%d %s", model.trackCount, getTrackCountNoun(model.trackCount))
```

Comment:
🍏 Еще можно использовать Kotlin tamplates: 
```suggestion
           "${model.trackCount} ${getTrackCountNoun(model.trackCount)}"
```